package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@storybook/addon-actions": "^5.3.19",
     "@storybook/vue": "^5.3.19",
+    "arg": "^4.1.3",
```

Comment:
Is this dependency being used? I don't see a reference to it in this Pull Request.

Reply:
This is used in https://github.com/nuxt-community/storybook/blob/master/src/cli.ts#L3 and has missed in `package.json`

Reply:
Oh, yes. I see the commit message. Thanks.